Branch-SDK/src/io/branch/referral/ServerRequestInitSession.java

```diff
@@ -71,6 +71,26 @@ public boolean handleBranchViewIfAvailable(ServerResponse resp) {
         return isBranchViewShowing;
     }
 
+    @Override
+    public void onRequestSucceeded(ServerResponse response, Branch branch) {
+        // Check for any Third party SDK for data handling
+        String appOrigin = "";
+        try {
+            if (response.getObject().has(Defines.Jsonkey.AppOrigin.getKey())) {
+                appOrigin = response.getObject().getString(Defines.Jsonkey.AppOrigin.getKey());
+            }
+            boolean isFabricEnabled = appOrigin.equals(Defines.Jsonkey.AppOriginTwitterFabric.getKey());
```

Comment:
With this check, people will only be able to use Branch with Fabric if they originally provisioned the Branch account through Fabric, right? If they already have a Branch account and then decide they wanted to use it with Fabric they'd be totally out of luck.

What I'd suggest here is dropping this check and pass the event to ```ExtendedAnswerProvider().provideData()```. The AnswersOptionalLogger will noop if the client doesn't have Fabric/Answers installed.